packages/core/src/utils/constants.js

```diff
@@ -33,7 +33,8 @@ export const colors = {
 export const delay = {
 	NOW: 100,
 	FAST: 200,
-	SLOW: 600
+	SLOW: 600,
+	CLOSE: 1500
```

Comment:
I think it would be better if we move this into the component itself.  We probably should get rid of this delay object all together soon, I don't think it brings that much of value to share these values between components.

Reply:
I was thinking it makes a lot of sense to have shared timing constants, but also noticed that most (all?) components just have hardcoded values for timeouts. The `delay` object is otherwise only used by the core, in which it is located, so I think we can safely keep the object. I will make my timeout use a local value. That is in alignment with other elements. If we ever want to make that shared constants, we should decide that when planning future development of the entire library.